lib/transports/mailhog.js

```diff
@@ -0,0 +1,23 @@
+var nodemailer = require('nodemailer');
+var debug = require('debug')('loopback-component-mailer');
+
+module.exports = function (options) {
+    const transporter = nodemailer.createTransport({
+        host: 'localhost',
+        port: 1025,
+    });
```

Comment:
Allowing these to come in from the options might be nice:
```js
const defaults = {
  host: 'localhost',
  port: 1025
}
const transportOptions = Object.assign({}, defaults, options)
const transporter = nodemailer.createTransport(transportOptions)
```
Then the consumer can set their own host and port values as needed

Reply:
Yep, nice idea - I have added this